packages/@okta/vuepress-site/guides/sign-into-web-app/get-user-info/aspnetcore/getuserinfo.md

```diff
@@ -0,0 +1,35 @@
+ASP.NET Core automatically populates `HttpContext.User` with the information Okta sends back about the user. You can check whether the user is logged in with `User.Identity.IsAuthenticated` in your actions or views, and see all of the user's claims in `User.Claims`.
+
+To access claims in your controllers use the `User` property which is attached to the `HttpContext` object:
```

Comment:
To access claims in your controllers, use the `User` property that is attached to the `HttpContext` object: